mvp-spec/ad-auction-example.md

```diff
@@ -991,3 +991,55 @@ graph LR;
 <!--partial-end-->
 </details>
 
+## Signatures
+
+In this section, we will focus on the signature to explain what exactly is signed at which moment. It is an example and based on [security-signatures.md](security-signatures.md)
+
+1. operator-signature-done-before-ad-auction-flow
```

Comment:
It's not obvious to me what these keywords are used for. Is it just a "name" to be used further down in the description, like line [1034](https://github.com/prebid/addressability-framework/pull/174/files#diff-a19822d50a17045e37c337af073fd3b41c8889e9b41daf39d1aac2e0cdb2d832R1034)?

In this case, why not just mention the step number? ("step 4").

And if it is something else (like a diagram to include?), then I guess it is missing.